### ISSUE TITLE
Combo box : The color contrast ratio of the combo box edit control is less than 3:1.

### ISSUE THREAD

**Issue by Sandhya-aluri** [microsoft/fluentui#17626]
### Environment Information

- **Package version(s)**: "@fluentui/react": "^7.123.4",
- **Browser and OS versions**: Chrome and EDGE , Windows 10

### Describe the issue:
The color contrast ratio of the combo box edit control is less than 3:1.

#### Actual behavior:
The color contrast ratio of the combo box edit control is less than 3:1.

#### Expected behavior:
The color contrast ratio of the combo box edit control  should be equal to or greater than 
3:1.

<img width="960" alt="Combobox" src="https://user-images.githubusercontent.com/48669559/113002668-a5074500-918f-11eb-86e0-55d3fbafcfe7.png">

**Comment by gouttierre** [comment]
@Sandhya-aluri - Thank you for filing this item.
@TristanWatanabe - Would you be able to confirm if color contrast ratio should be 3:1?
